src/index.html

```diff
@@ -5,8 +5,328 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class = "header">
+      <a href = "#home" class = "header__logo">
+        <img src = "images/logo.png" alt = "Moyo_logo">
+      </a>
```

Comment:
Let's remove spaces between the attribute name and it's value
```suggestion
      <a href="#home" class="header__logo">
        <img src="images/logo.png" alt ="Moyo_logo">
      </a>
```

Reply:
But maybe do it in the next PR's